po/es.po

```diff
@@ -10,6 +10,7 @@
 # Guille <willelopz+weblate@gmail.com>, 2021.
 # Adolfo Jayme Barrientos <fitojb@ubuntu.com>, 2021.
 # Francisco Serrador <fserrador@gmail.com>, 2021, 2022.
+# Cow <javier.fserrador@gmail.com>, 2022.
```

Comment:
@cyphra I see at least 3 different aliases for one email address. That is confusing—not only for the weblate bot.

Tell me when you fixed your weblate account's email and I can apply it on the different es.po files in the Gnucash project.